test/common/stats/thread_local_store_test.cc

```diff
@@ -1958,5 +1959,193 @@ TEST_F(HistogramThreadTest, ScopeOverlap) {
   store_->histogramFromString("histogram_after_shutdown", Histogram::Unit::Unspecified);
 }
 
+enum class EnableIncludeHistograms { No = 0, Yes };
+class HistogramParameterisedTest : public HistogramTest,
+                                   public ::testing::WithParamInterface<EnableIncludeHistograms> {
+public:
+  HistogramParameterisedTest() { local_info_.node_.set_cluster(""); }
+
+protected:
+  void SetUp() override {
```

Comment:
any particular reason not to put this in the ctor? Adding code comments would be fine.

Reply:
This needs store_ which is created in SetUp for the  base class. Added comments.